quantities/common/ITransform.cs

```diff
@@ -2,6 +2,5 @@ namespace Quantities;
 
 public interface ITransform
```

Comment:
Here is the resultant change on the interface that all units (that are not si units) must implement.

Note that the ´FromSi´ method could be deleted. As thanks to the generalised way we record the conversions, the library can compute the inverse operations.

This reduces the likelihood for missmatching "To" and "From" conversions to zero :-)